lldb/test/API/lang/swift/clangimporter/rewrite_clang_paths/TestSwiftRewriteClangPaths.py

```diff
@@ -124,7 +124,7 @@ def dotest(self, remap):
             self.assertEqual(found_iquote, 2)
             self.assertEqual(found_i1, 2)
             self.assertEqual(found_i2, 2)
-            self.assertEqual(found_f, 4)
```

Comment:
Seems like deduplication and args joining has taken some effect here:
[after.log](https://github.com/apple/llvm-project/files/5062735/after.log)
[before.log](https://github.com/apple/llvm-project/files/5062736/before.log)
There was a duplicated `-F` arg that was successfully uniqued.